apps/console/src/features/applications/components/settings/sign-on-methods/sign-in-method-customization.tsx

```diff
@@ -558,6 +573,51 @@ export const SignInMethodCustomization: FunctionComponent<SignInMethodCustomizat
         );
     };
 
+    const SMSOTPConflictMessage = () => {
+
+        const FIRST_ENTRY: number = 0;
+        const { idpList } = smsValidationResult;
+        const moreThan1IdP: boolean = idpList.length > 1;
```

Comment:
`FIRST_ENTRY` and `moreThan1IdP` are unnecessary variables IMO. We can directly use the assigned expressions (`0` and `idpList.length> 1`) in the respective places to improve readability. WDYT?

```suggestion
        const { idpList } = smsValidationResult;
```

Reply:
Your suggestion looks good to me.